src/lib/action.ts

```diff
@@ -213,19 +225,51 @@ export default class ActionHandler {
     }
   };
 
-  requestTeX = async (): Promise<void> => {
+  requestTeX = async (): Promise<
+    "success" | "no latex entered" | "invalid latex"
```

Comment:
nit: we should be consistent with capitalization

(while we're at it, we should change it to `Enter LaTeX source:` probably)